### ISSUE TITLE
FloatingIP module doesn't respect computed values

### ISSUE THREAD

**Issue by dev00** [terraform-providers/terraform-provider-openstack#842]
### Terraform Version
```
Terraform v0.12.6
+ provider.openstack v1.21.1
```
### Affected Resource(s)
- openstack_networking_network_v2

### Terraform Configuration Files
```hcl
resource "openstack_networking_floatingip_v2" "test_fip" {
  pool = "${var.external_net}"
}
```

### Debug Output
```
  # module.ecs.openstack_networking_floatingip_v2.test_fip must be replaced
-/+ resource "openstack_networking_floatingip_v2" "test_fip" {
      ~ address    = "192.168.10.13" -> (known after apply)
      ~ all_tags   = [] -> (known after apply)
      - dns_domain = "reverse.open-telekom-cloud.com." -> null # forces replacement
      - dns_name   = "ecs-192-168-10-13" -> null # forces replacement
      ~ fixed_ip   = "10.10.10.10" -> (known after apply)
      ~ id         = "70427390-70d2-4efb-8e72-b356c1339342" -> (known after apply)
        pool       = "admin_external_net"
      ~ port_id    = "3a00e9e9-d938-4bb7-93de-30e4ac9ceebe" -> (known after apply)
      + region     = (known after apply)
      - tags       = [] -> null
      ~ tenant_id  = "a_secret_id" -> (known after apply)
    }
```

### Expected Behavior
When Openstack decides the field `dns_domain`, `dns_name` and `tags` are should be filled with default values, terraform should respect that.

### Actual Behavior
Terraform tries to overwrite said default values, resulting in unneeded changes.

### Steps to Reproduce
1. `get Openstack with dns plugin enabled`
2. `get floating ip via tf openstack`
3. `apply twice`

### Important Factoids
This was tested on OTC, which provides a (mostly) Openstack compliant API. As this behaviour is not specific to OTC, but rather a question how default values in general are handled, I would like to see a similar change like the #837

**Comment by kayrus** [comment]
Wasn't it fixed in #837?

**Comment by dev00** [comment]
@kayrus Oh, just saw you also changed the floatingip resource. I'm sorry, you're right...
